libpod/networking_pasta_linux.go

```diff
@@ -7,105 +7,13 @@
 
 package libpod
 
-import (
-	"fmt"
-	"os/exec"
-	"strings"
-
-	"github.com/sirupsen/logrus"
-)
-
-const (
-	pastaBinaryName = "passt"
```

Comment:
The new libnetwork/pasta/pasta.go calls it `pasta` and is more consistent in that use (vs line 29 below). On my f38 laptop I see `/usr/bin/pasta -> passt`; also on f37. I'm assuming this is a deliberate change, and that the `passt` RPM will forevermore guarantee this symlink, but I don't like silent assumptions so I'm pointing it out.

Reply:
I made it an extra commit in c/common after moving the code: https://github.com/containers/common/pull/1506/commits/d9ee72cee5eebc1fbe403e972bc89bc30fc59253
Yes it is always a symlink but we only ever use pasta so it doesn't make sense to call it passt in info IMO.